File: cpp/media_idades.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    int idade, npessoas = 0;
    double soma = 0, media;

    cout << "Digite as idades:" << endl;
    cin >> idade;

    if(idade < 0){
        cout << "IMPOSSIVEL CALCULAR" << endl;
    }
    else{
        while(idade >= 0){
            soma = soma + idade;
            npessoas++;
            cin >> idade;
        }

        media = soma / npessoas;

        cout << fixed << setprecision(2);
        cout << "MEDIA = " << media << endl;
    }

	return 0;
}

File: python/media_idades.py
idade: int; npessoas: int
soma: float; media: float

print("Digite as idades:")
idade = int(input())

if idade < 0:
	print("IMPOSSIVEL CALCULAR")
else:
	soma = 0
	npessoas = 0
	while idade >= 0:
		soma = soma + idade
		npessoas = npessoas + 1
		idade = int(input())

	media = soma / npessoas

	print(f"MEDIA = {media:.2f}")